intro_to_circleci.md

```diff
@@ -606,9 +583,14 @@ workflows:
       - run_specs:
           requires:
             - lintering
-
 ```
+## Plan Overview
+
+In the CircleCi have a different plans for accounts.If you don’t have enough machine power (CPU, memory) you can configure them in the tab `setting -> plan overview` and upgrade them to **PERFORMANCE**.
```

Comment:
Sorry, I think your text has grammar errors, would write smth like this:

There are different plans for accounts in CircleCI. If you need more machine power (CPU, memory) you can upgrade your plan to **PERFORMANCE** in the `settings -> plan overview` tab.